.eslintrc.js

```diff
@@ -392,6 +400,7 @@ module.exports = {
 			excludedFiles: [
 				'packages/components/src/utils/colors-values.js',
 				'packages/components/src/theme/**',
+				'**/*.@(native|ios|android).js',
```

Comment:
Not excluding these mobile files was an oversight.

Reply:
👾